sale_delivery_state/__manifest__.py

```diff
@@ -5,13 +5,14 @@
 {
     "name": "Sale delivery State",
     "summary": "Show the delivery state on the sale order",
-    "version": "14.0.1.1.0",
+    "version": "15.0.1.0.0",
     "category": "Product",
     "website": "https://github.com/OCA/sale-workflow",
     "author": "Akretion, Odoo Community Association (OCA)",
     "license": "AGPL-3",
     "installable": True,
     "depends": ["sale"],
+    "external_dependencies": {"python": ["mock"]},
```

Comment:
please remove this, it's not a functional dependency

Reply:
also `mock` is a built-in lib, no need to declare its dependency :wink:

Reply:
Thank for you review, i updated code according your suggestions